scripts/versions.mjs

```diff
@@ -29,7 +29,7 @@ await Promise.allSettled(
 			fileToESMWrapperCall('utils', version),
 
 			// Shortcuts
-			fileToESMWrapperCall('shortcuts', version),
+			fileToESMWrapperCall('owo/..', version),
```

Comment:
What's this

Reply:
Joke: OwO
Actual: The script expects a path to the file to convert, and you cannot do `./../` sadly (I tried). So, to alleviate, I gave it a non-existent path just so it can go to the root folder where the file is

Reply:
Then use [fileURLToPath](https://nodejs.org/dist/latest-v15.x/docs/api/url.html#url_url_fileurltopath_url), it's less hacky and should work if this is just expecting a path:

```javascript
const __filename = fileURLToPath(import.meta.url);
const __dirname = dirname(__filename);
const rootDir = join(__dirname, '..');
```

If that doesn't, you should refactor the path checking to check whether [it's absolute](https://nodejs.org/dist/latest-v15.x/docs/api/path.html#path_path_isabsolute_path) or not.

Reply:
Done in https://github.com/discordjs/discord-api-types/pull/113/commits/1fa258ed3f36d4d786fb4ebca0a55b89a7d5b788